specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-09-01-preview/apimgatewayConfigConnections.json

```diff
@@ -0,0 +1,348 @@
+          "201": {
+            "description": "The gateway config connection was successfully created.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      },
+      "delete": {
+        "tags": [
+          "ApiManagementGatewayConfigConnection"
+        ],
+        "operationId": "ApiManagementGatewayConfigConnection_Delete",
+        "description": "Deletes an existing API Management gateway config connection.",
+        "x-ms-examples": {
+          "ApiManagementGatewayDeleteGateway": {
+            "$ref": "./examples/ApiManagementDeleteGatewayConfigConnection.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/GatewayNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ConfigConnectionIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "The gateway config connection delete request was accepted.",
+            "schema": {
+              "$ref": "#/definitions/ApiManagementGatewayConfigConnectionResource"
+            },
+            "headers": {
+              "location": {
+                "description": "Location header",
+                "type": "string"
+              }
+            }
+          },
+          "204": {
+            "description": "The gateway config connection does not exist."
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    }
+  },
+  "definitions": {
+    "GatewayConfigConnectionBaseProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```

Comment:
>provisioningState [](http://example.com/codeflow?start=9&length=17)

please define the ProvisioningState enum for your resource type

Reply:
What's the difference between Active, Created, and Succeeded? And what's the difference between Stopped, Failed, and Canceled?

Reply:
Per my other comment you can't add to the 3 ARM terminal states (Succeeded, Failed, Canceled). Active, Created, and Stopped appear to be additional terminal states. If they are something else, please explain.